### PR TITLE
Misc release_tool updates

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

File: extra/release_tool.py [hunk 1]
```diff
@@ -2389,8 +2400,11 @@ def do_release(release_state_file):
     if input.startswith("Y") or input.startswith("y"):
         refresh_repos(state)
 
-    for repo in sorted(Component.get_components_of_type("git"), key=repo_sort_key):
-        determine_version_to_include_in_release(state, repo)
+    repos = sorted(Component.get_components_of_type("git"), key=repo_sort_key)
+    while len(repos) > 0:
+        repo = repos.pop(0)
+        if not determine_version_to_include_in_release(state, repo):
+            repos.append(repo)
```
Comment (kacf) [inline extra/release_tool.py:2411]: Clever! I had to read this a few times to understand that it keeps looping until all entries are decided.

File: other-components-docker.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# This file lists software components, and their versions, that are part of a
+# Mender release, have Docker containers, but are not part a default Mender
+# installation (have no docker-compose file).
+services:
+    mtls-ambasador:
```
Comment (kacf) [inline other-components-docker.yml:5]: `s/ambasador/ambassador/`

**Review by kacf — APPROVED** [review]
Watch out for the typo below! I trust you will fix, so pre-approving.

File: other-components-docker.yml (hunk 2 above)
Comment (lluiscampos) [reply other-components-docker.yml:5]: Thanks. I'll check for more occurrences of this typo.

**Pull request merged**